core/src/components/datetime/datetime-interface.ts

```diff
@@ -23,3 +23,5 @@ export interface DatetimeParts {
 }
 
 export type DatetimePresentation = 'date-time' | 'time-date' | 'date' | 'time' | 'month' | 'year' | 'month-year';
+
+export type SelectedDatesLabelFormatter = (datesSelected: number) => string;
```

Comment:
~ Should we tweak the naming of `dateSelected` to represent a number? e.g.: `selectedDateCount`, `numberOfDates`, `count`, etc. 

With the current naming it may be mistaken as the array of dates, until they interact with the type.